web/src/javascript/adminApp.js

```diff
@@ -4,5 +4,5 @@ const seedButton = document.getElementById("seedButton");
 seedButton.onclick = () => {
   sendAdminRequest("player", "post", {seed: true})
     .then(() => alert("The DB was seeded"))
-    .catch(e => alert(e));
+    .catch(() => window.location.href = "/web/adminLogin.html");
```

Comment:
- Could we put this logic in `sendAdminRequest`? It's the function that all calls is using.
- Also, could we only redirect on `403`. We can have errors that don't mean that we have a bad password.